spec/customer_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'restaurant'
+
+describe 'customer' do
+
+  subject {Customer.new}
+
+  it 'creates a new customer' do
+    expect(subject).to be_kind_of Customer
+  end
+
+    it 'has a order sheet' do
+    expect(subject.order).to be_kind_of Array
+  end
```

Comment:
Lint/BlockAlignment: `end` at 13, 2 is not aligned with `it 'has a order sheet' do` at 11, 4